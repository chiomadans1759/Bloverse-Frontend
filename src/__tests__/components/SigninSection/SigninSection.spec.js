import { mount, RouterLinkStub  } from '@vue/test-utils'
import SigninComponent from '@/components/SigninSection/Signin.vue';

describe('SigninComponent Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(SigninComponent, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    });
    expect(wrapper.find(RouterLinkStub).props().to).toBe('/creators/apply')
    expect(wrapper.element).toMatchSnapshot();
  })
});
