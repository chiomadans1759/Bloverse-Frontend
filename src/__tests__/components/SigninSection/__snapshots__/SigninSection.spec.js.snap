// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninComponent Component should render correctly 1`] = `
<div
  class="mainSection"
>
  <h2
    class="welcome-text"
  >
    Welcome back
  </h2>
   
  <br />
  <br />
   
  <h2
    class="intro-text"
  >
    Login With
  </h2>
   
  <br />
   
  <div
    class="social-login"
  >
    <div>
      <button
        class="defaultButton"
      >
        <img
          src="@/assets/facebook-logo-light.svg"
        />
      </button>
    </div>
     
    <br />
     
    <div>
      <button
        class="defaultButton"
      >
        <img
          height="25"
          src="@/assets/google-text-logo.png"
          width="60"
        />
      </button>
    </div>
  </div>
   
  <br />
   
  <div
    class="grid-container"
  >
    <div
      class="grid-item"
    >
      <hr
        class="hr-text"
      />
    </div>
     
    <div
      class="middle-grid-item"
    >
      <p>
        Or
      </p>
    </div>
     
    <div
      class="grid-item"
    >
      <hr
        class="hr-text"
      />
    </div>
  </div>
   
  <br />
   
  <div
    class="form-section"
  >
    <input
      class="defaultClass text-input"
      id="email"
      name="email"
      placeholder="Email"
      type="email"
    />
     
    <br />
    <br />
     
    <input
      class="defaultClass text-input"
      id="password"
      name="password"
      placeholder="Password"
      type="password"
    />
     
    <br />
    <br />
     
    <p
      class="forgot-password-link"
    >
      <a>
        Forgot password ?
      </a>
    </p>
     
    <br />
     
    <div>
      <button
        class="defaultButton primary-button"
      >
        Login
      </button>
    </div>
     
    <br />
    <br />
     
    <div>
      <button
        class="defaultButton secondary-button"
      >
        <h2
          class="new-bloverse-text"
        >
          New to bloverse? 
        
          <a
            class="signup-text"
          >
            Sign Up
          </a>
        </h2>
      </button>
    </div>
  </div>
   
  <br />
  <br />
   
  <h2
    class="creator-text"
  >
    Content creator? 
    
    <a
      class="router-link apply-now-text"
    >
      
      Apply
    
    </a>
  </h2>
</div>
`;
